src/nixpacks/plan/generator.rs

```diff
@@ -165,11 +165,20 @@ impl<'a> NixpacksBuildPlanGenerator<'a> {
             .or(env_build_cmd)
             .or(build_phase.cmd);
 
+        // Release process type
+        if let Some(release_cmd) = self.get_procfile_release_cmd(app)? {
```

Comment:
I may be missing something, but why is the release command being used as the build command?

Reply:
Release is run after build but before start ([source](https://devcenter.heroku.com/articles/release-phase)). And Cooper was [against adding](https://canary.discord.com/channels/713503345364697088/982408838063988756) new phases.